bindings/generator/api/common.py

```diff
@@ -199,3 +199,15 @@ class UserProfile(Variant):
     Admin = VariantItemUnit
     Standard = VariantItemUnit
     Outsider = VariantItemUnit
+
+
+async def build_backend_addr(
```

Comment:
```suggestion
def build_backend_addr(
```

Don't need to be async